packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -5,17 +5,16 @@ import React, {
   useMemo,
   useState,
 } from 'react';
+import dynamic from 'next/dynamic';
```

Comment:
What are we loading dynamic?

Reply:
This is a lint fix. The current branch is failing on lint so I ran a lint fix. Basically this import was a bit down before and the lint moved it up here